common/lib/xmodule/xmodule/js/js_test.yml

```diff
@@ -59,6 +59,7 @@ lib_paths:
     - public/js/split_test_staff.js
     - common_static/js/src/accessibility_tools.js
     - common_static/js/vendor/moment.min.js
+    - common_static/js/vendor/draggabilly.pkgd.js
```

Comment:
Also add draggabilly to the RequireJS config in spec/main_requirejs.js (which is only used for xmodule tests).